azure/image/create-user.sh

```diff
@@ -0,0 +1,7 @@
+ssh_username="peerpod"
+
+if [ -n "$1" ]; then
+    ssh_username=$1
+fi
+
+sudo useradd -m -s /bin/bash "${ssh_username}"
```

Comment:
```suggestion
```

Reply:
IMHO the default username setting here is redundant it is already set in the packer variables file.